lib/exchange/exolix/exolix_exchange_provider.dart

```diff
@@ -0,0 +1,293 @@
+import 'dart:convert';
+import 'package:cake_wallet/exchange/trade_not_found_exeption.dart';
+import 'package:http/http.dart';
+import 'package:cake_wallet/.secrets.g.dart' as secrets;
+import 'package:cw_core/crypto_currency.dart';
+import 'package:cake_wallet/exchange/exchange_pair.dart';
+import 'package:cake_wallet/exchange/exchange_provider.dart';
+import 'package:cake_wallet/exchange/limits.dart';
+import 'package:cake_wallet/exchange/trade.dart';
+import 'package:cake_wallet/exchange/trade_request.dart';
+import 'package:cake_wallet/exchange/trade_state.dart';
+import 'package:cake_wallet/exchange/exolix/exolix_request.dart';
+import 'package:cake_wallet/exchange/exchange_provider_description.dart';
+
+class ExolixExchangeProvider extends ExchangeProvider {
+  ExolixExchangeProvider() : super(pairList: _supportedPairs());
+
+  static final apiKey = secrets.exolixApiKey;
+  static const apiBaseUrl = 'https://exolix.com/api';
```

Comment:
this was causing an issue since you are trying to create an https uri (`Uri.https(...`) while use a base url that already includes `http`

also `/api` should not be in the base url, instead it should be in the below paths

Reply:
https://exolix.com/developers